internal/service/advancedcluster/model_advanced_cluster_test.go

```diff
@@ -21,6 +23,128 @@ var (
 	advancedClusters = []admin.AdvancedClusterDescription{{StateName: conversion.StringPtr("NOT IDLE")}}
 )
 
+func TestFlattenReplicationSpecs(t *testing.T) {
+	var (
+		regionName         = "EU_WEST_1"
+		providerName       = "AWS"
+		expectedID         = "id1"
+		unexpectedID       = "id2"
+		expectedZoneName   = "z1"
+		unexpectedZoneName = "z2"
+		regionConfigAdmin  = []admin.CloudRegionConfig{{
+			ProviderName: &providerName,
+			RegionName:   &regionName,
+		}}
+		regionConfigTf = map[string]any{
+			"provider_name": "AWS",
+			"region_name":   regionName,
+		}
+		regionConfigTfDiffZone = map[string]any{
+			"provider_name": "AWS",
+			"region_name":   regionName,
+			"zone_name":     unexpectedZoneName,
+		}
+		admin1     = admin.ReplicationSpec{Id: &expectedID, ZoneName: &expectedZoneName, RegionConfigs: &regionConfigAdmin}
```

Comment:
similar to comment below, would replace numeration with more significant name so it is easier to follow test case definitions

Reply:
Renamed in 9425ff49b8b3e0c3b0dad38fc44160d40e88033b
WDYT?